### PR TITLE
Updating the Docker image readies.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

File: java/images/rhel/README.md [hunk 1]
```diff
@@ -21,15 +21,45 @@ The following environment variables can be used to influence the behaviour of th
...
-during building and running.
+* **JAVA_APP_DIR** the directory where the application resides. All paths in your application are relative to this directory.
+* **JAVA_LIB_DIR** directory holding the Java jar files as well an optional `classpath` file which holds the classpath. Either as a single line classpath (colon separated) or with jar files listed line-by-line. If not set **JAVA_LIB_DIR** is the same as **JAVA_APP_DIR**.
+* **JAVA_OPTIONS** options to add when calling `java`
+* **JAVA_MAX_MEM_RATIO** is used when no `-Xmx` option is given in `JAVA_OPTIONS`. This is used to calculate a default maximal Heap Memory based on a containers restriction. If this script is used in a Docker container without any memory constraints for the container then this option has no effect. If there is a memory constraint then `-Xmx` is set to a ratio of the container available memory as set here. The default is 85 which means 85% of the available memory is used as an upper boundary. You can skip this mechanism by setting this value to 0 in which case no `-Xmx` option is added on behalf od this script.
```
Comment (maschmid) [inline java/images/rhel/README.md]: I believe it is 50% by default now...

Comment (rhuss) [inline java/images/rhel/README.md]: Yes, its 50%. However, this should have been updated in https://github.com/rhuss/run-java-sh as it is included by fishpepper from there.

Comment (rhuss) [inline java/images/rhel/README.md]: In fact, https://github.com/rhuss/run-java-sh/blob/master/fish-pepper/run-java-sh/readme.md contains already the corrected size, so maybe one just need to re-run fish-pepper ?

Comment (chirino) [inline java/images/rhel/README.md]: you need to PR those changes up.

**Comment by chirino** [comment]
[merge]

**Comment by chirino** [comment]
[lgtm]

**Pull request merged**